phylogenetic/defaults/clade-i/config.yaml

```diff
@@ -0,0 +1,58 @@
+reference: "defaults/reference.fasta"
+genome_annotation: "defaults/genome_annotation.gff3"
+genbank_reference: "defaults/reference.gb"
+include: "defaults/clade-i/include.txt"
+clades: "defaults/clades.tsv"
+lat_longs: "defaults/lat_longs.tsv"
+auspice_config: "defaults/clade-i/auspice_config.json"
+description: "defaults/description.md"
+tree_mask: "defaults/tree_mask.tsv"
+
+# Use `accession` as the ID column since `strain` currently contains duplicates¹.
+# ¹ https://github.com/nextstrain/mpox/issues/33
+strain_id_field: "accession"
+display_strain_field: "strain"
+
+build_name: "clade-i"
+auspice_name: "mpox_clade-I"
+
+filter:
+  min_date: 1900
+  min_length: 100000
+  exclude_where: 'clade!=I'
```

Comment:
If the Nextclade clade definitions change to include clade-Ib (on the mpxv dataset, as that's used to assign clades AFAICT) then we need to update this.